### ISSUE TITLE
fix: startDate needs to be Date.Now() after login

### ISSUE THREAD

**Issue by daborton** [brown-ccv/neuro-task-starter#128]
Currently, startDate refers to the app window creation, not the start of the experiment, which can be a very different time. Further, time_elapsed from jsPsych is relative to the start of the experiment, not the creation of the app window. This is problematic when synchronizing to other data collected in parallel to the task data.

**Comment by RobertGemmaJr** [comment]
Duplicate of #126
